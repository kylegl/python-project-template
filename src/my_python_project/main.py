def main():
    """Entry Point for the project."""
    print("Hello, World!")
    return "Hello, World!"
