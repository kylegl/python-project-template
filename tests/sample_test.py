from src.my_python_project.main import main

def test_main(snapshot):
    actual = main()

    assert actual == snapshot
