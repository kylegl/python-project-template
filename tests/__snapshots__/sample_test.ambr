# serializer version: 1
# name: test_main
  'Hello, World!'
# ---
